src/main/java/net/snowflake/client/core/HttpUtil.java

```diff
@@ -372,12 +407,17 @@ public static CloseableHttpClient buildHttpClient(
               new UsernamePasswordCredentials(key.getProxyUser(), key.getProxyPassword());
           AuthScope authScope = new AuthScope(key.getProxyHost(), key.getProxyPort());
           CredentialsProvider credentialsProvider = new BasicCredentialsProvider();
+          logger.debug("Using user: {} for proxy host: {}, port: {}",
+                  key.getProxyUser(),
```

Comment:
do we want to log that proxy pass is provided?

Reply:
Right, done